internal/adminapi/endpoints.go

```diff
@@ -3,48 +3,43 @@ package adminapi
 import (
 	"context"
 	"fmt"
-	"strings"
 
 	discoveryv1 "k8s.io/api/discovery/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/selection"
 	k8stypes "k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	cfgtypes "github.com/kong/kubernetes-ingress-controller/v3/internal/manager/config/types"
 )
 
 // DiscoveredAdminAPI represents an Admin API discovered from a Kubernetes Service.
+// For field Address use format https://<podIP>:<port>, and for field TLSServerName
+// use format <podIPDashes>.<serviceName>.<namespace>.svc, where podIPDashes
+// is the IP address separated by dashes instead of dots.
```

Comment:
```suggestion
// The following formats are used for the fields:
// - Address: `https://<podIP>:<port>`
// - TLSServerName: `<podIPDashes>.<serviceName>.<namespace>.svc`, where podIPDashes
//   is the IP address separated by dashes instead of dots.
```